packages/coq-waterproof/coq-waterproof.2.1.1+8.18/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "Jim Portegies <j.w.portegies@tue.nl>"
+authors: [
+  "Jelle Wemmenhove"
+  "Balthazar Pathiachvili"
+  "Cosmin Manea"
+  "Lulof Pirée"
+  "Adrian Vrămuleţ"
+  "Tudor Voicu"
+  "Jim Portegies <j.w.portegies@tue.nl>"
+]
+
+synopsis: "Coq proofs in a style that resembles non-mechanized mathematical proofs"
+description: """
+The coq-waterproof library allows you to write Coq proofs in a style that resembles non-mechanized mathematical proofs.
+Mathematicians unfamiliar with the Coq syntax are able to read the resulting proof scripts.
+"""
+
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/impermeable/coq-waterproof"
+dev-repo: "git+https://github.com/impermeable/coq-waterproof.git"
+bug-reports: "https://github.com/impermeable/coq-waterproof/issues"
+
+depends: [
+  "ocaml" {>= "4.09.0"}
+  "coq" {>= "8.18" & < "8.19"}
+  "dune" {>= "3.6."}
+]
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs "@install"]
```

Comment:
This is the standard dune invocation, it also generates the `.install` file that is used by opam to install the package, so the `install` stanza is not needed